{"@t":"2022-09-15T04:44:04.4282367Z","@mt":"Entity Framework Core {version} initialized '{contextType}' using provider '{provider}:{providerVersion}' with options: {options}","version":"6.0.8","contextType":"ApplicationDbContext","provider":"Microsoft.EntityFrameworkCore.SqlServer","providerVersion":"6.0.8","options":"None","EventId":{"Id":10403,"Name":"Microsoft.EntityFrameworkCore.Infrastructure.ContextInitialized"},"SourceContext":"Microsoft.EntityFrameworkCore.Infrastructure","ActionId":"44380638-763f-41f0-ba9c-a977773a6792","ActionName":"TweetApp.Backend.Controllers.UsersController.Login (TweetApp.Backend)","RequestId":"0HMKM5V4D6RBF:00000003","RequestPath":"/api/v1/tweets/login","ConnectionId":"0HMKM5V4D6RBF"}
{"@t":"2022-09-15T04:44:21.3151476Z","@mt":"An error occurred using the connection to database '{database}' on server '{server}'.","@l":"Error","database":"tweetappapi","server":"tcp:tweetapp-server-atman.database.windows.net,1433","EventId":{"Id":20004,"Name":"Microsoft.EntityFrameworkCore.Database.Connection.ConnectionError"},"SourceContext":"Microsoft.EntityFrameworkCore.Database.Connection","ActionId":"44380638-763f-41f0-ba9c-a977773a6792","ActionName":"TweetApp.Backend.Controllers.UsersController.Login (TweetApp.Backend)","RequestId":"0HMKM5V4D6RBF:00000003","RequestPath":"/api/v1/tweets/login","ConnectionId":"0HMKM5V4D6RBF"}
{"@t":"2022-09-15T04:44:21.3723569Z","@mt":"An exception occurred while iterating over the results of a query for context type '{contextType}'.{newline}{error}","@l":"Error","@x":"System.InvalidOperationException: An exception has been raised that is likely due to a transient failure. Consider enabling transient error resiliency by adding 'EnableRetryOnFailure' to the 'UseSqlServer' call.\r\n ---> Microsoft.Data.SqlClient.SqlException (0x80131904): Database 'tweetappapi' on server 'tweetapp-server-atman.database.windows.net' is not currently available.  Please retry the connection later.  If the problem persists, contact customer support, and provide them the session tracing ID of '{C8A44902-D106-4228-9152-F7C68C3B33E7}'.\r\n   at Microsoft.Data.ProviderBase.DbConnectionPool.CheckPoolBlockingPeriod(Exception e)\r\n   at Microsoft.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)\r\n   at Microsoft.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)\r\n   at Microsoft.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)\r\n   at Microsoft.Data.ProviderBase.DbConnectionPool.WaitForPendingOpen()\r\n--- End of stack trace from previous location ---\r\n   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.OpenInternalAsync(Boolean errorsExpected, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.OpenInternalAsync(Boolean errorsExpected, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.OpenAsync(CancellationToken cancellationToken, Boolean errorsExpected)\r\n   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Query.Internal.SingleQueryingEnumerable`1.AsyncEnumerator.InitializeReaderAsync(AsyncEnumerator enumerator, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)\r\nClientConnectionId:902c0c48-ef4f-4869-8a58-70813c32a181\r\nError Number:40613,State:1,Class:20\r\n   --- End of inner exception stack trace ---\r\n   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Query.Internal.SingleQueryingEnumerable`1.AsyncEnumerator.MoveNextAsync()","contextType":"TweetApp.Backend.Repository.ApplicationDbContext","newline":"\r\n","error":"System.InvalidOperationException: An exception has been raised that is likely due to a transient failure. Consider enabling transient error resiliency by adding 'EnableRetryOnFailure' to the 'UseSqlServer' call.\r\n ---> Microsoft.Data.SqlClient.SqlException (0x80131904): Database 'tweetappapi' on server 'tweetapp-server-atman.database.windows.net' is not currently available.  Please retry the connection later.  If the problem persists, contact customer support, and provide them the session tracing ID of '{C8A44902-D106-4228-9152-F7C68C3B33E7}'.\r\n   at Microsoft.Data.ProviderBase.DbConnectionPool.CheckPoolBlockingPeriod(Exception e)\r\n   at Microsoft.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)\r\n   at Microsoft.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)\r\n   at Microsoft.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)\r\n   at Microsoft.Data.ProviderBase.DbConnectionPool.WaitForPendingOpen()\r\n--- End of stack trace from previous location ---\r\n   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.OpenInternalAsync(Boolean errorsExpected, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.OpenInternalAsync(Boolean errorsExpected, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.OpenAsync(CancellationToken cancellationToken, Boolean errorsExpected)\r\n   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Query.Internal.SingleQueryingEnumerable`1.AsyncEnumerator.InitializeReaderAsync(AsyncEnumerator enumerator, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)\r\nClientConnectionId:902c0c48-ef4f-4869-8a58-70813c32a181\r\nError Number:40613,State:1,Class:20\r\n   --- End of inner exception stack trace ---\r\n   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Query.Internal.SingleQueryingEnumerable`1.AsyncEnumerator.MoveNextAsync()","EventId":{"Id":10100,"Name":"Microsoft.EntityFrameworkCore.Query.QueryIterationFailed"},"SourceContext":"Microsoft.EntityFrameworkCore.Query","ActionId":"44380638-763f-41f0-ba9c-a977773a6792","ActionName":"TweetApp.Backend.Controllers.UsersController.Login (TweetApp.Backend)","RequestId":"0HMKM5V4D6RBF:00000003","RequestPath":"/api/v1/tweets/login","ConnectionId":"0HMKM5V4D6RBF"}
{"@t":"2022-09-15T04:44:21.8141227Z","@mt":"Invalid username or password","@l":"Error","SourceContext":"TweetApp.Backend.Controllers.UsersController","ActionId":"44380638-763f-41f0-ba9c-a977773a6792","ActionName":"TweetApp.Backend.Controllers.UsersController.Login (TweetApp.Backend)","RequestId":"0HMKM5V4D6RBF:00000003","RequestPath":"/api/v1/tweets/login","ConnectionId":"0HMKM5V4D6RBF"}
{"@t":"2022-09-15T04:44:21.8230006Z","@mt":"HTTP {RequestMethod} {RequestPath} responded {StatusCode} in {Elapsed:0.0000} ms","@r":["17411.8427"],"RequestMethod":"POST","RequestPath":"/api/v1/tweets/login","StatusCode":200,"Elapsed":17411.8427,"SourceContext":"Serilog.AspNetCore.RequestLoggingMiddleware","ActionId":"44380638-763f-41f0-ba9c-a977773a6792","ActionName":"TweetApp.Backend.Controllers.UsersController.Login (TweetApp.Backend)","RequestId":"0HMKM5V4D6RBF:00000003","ConnectionId":"0HMKM5V4D6RBF"}
